models/concerns/validate_data.rb

```diff
@@ -114,6 +138,12 @@ def check_vehicles(periodic_heuristic)
         }
       end
 
+      unless schedule || v[:sequence_timewindows].to_a.empty?
+        raise OptimizerWrapper::UnsupportedProblemError.new(
+          'There can only be one timewindow for vehicles when no schedule is provided'
```

Comment:
May be something more explicit (because this message is not enough if the file has only one timewindow in sequence_timewindows already).

`:sequence_timewindows` parameter is available only for problems with `:schedule` 

of something like that.